test/gate/printer/graphml_test.cpp

```diff
@@ -0,0 +1,13 @@
+#include "gate/printer/graphml.h"
+#include "gate/model/gnet_test.h"
+
+#include "gtest/gtest.h"
+
+using namespace eda::printer::graphMl;
+
+TEST(toGraphMlTest,all) {
+  GNet test = *makeRand(5, 5);
+  std::cout << test;
+  //Выводит не то, что надо, так как оператор вывода для GNet уже перегружен, 
```

Comment:
Remove this comment. Just mark your new printer with //FIXME: there is another operator<<